samples/Experimental/VulkanRender/CMakeLists.txt

```diff
@@ -3,29 +3,12 @@ set(SOURCES
         )
 
 add_executable(VulkanRender ${SOURCES})
-add_dependencies(VulkanRender Dotnet Resources)
+add_dependencies(VulkanRender Resources)
 target_link_libraries(VulkanRender
         PRIVATE
         Engine
         )
 
-add_custom_command(
-        TARGET VulkanRender POST_BUILD
-        COMMAND ${CMAKE_COMMAND} -E copy_directory
-        $<TARGET_FILE_DIR:Dotnet>
-        $<TARGET_FILE_DIR:VulkanRender>/dotnet
-)
-
-#this is pure hacky hotfix goodness. We need to figure out a better way to do this in the future.
-if(WIN32)
-    add_custom_command(
-            TARGET VulkanRender POST_BUILD
-            COMMAND ${CMAKE_COMMAND} -E copy
-            $<TARGET_FILE_DIR:VulkanRender>/dotnet/nethost.dll
-            $<TARGET_FILE_DIR:VulkanRender>
-    )
-endif()
-
```

Comment:
Much confuse why VulkanRender is being touched here.

Reply:
Shouldn't be including Ink when not required - this breaks configuring if ink is defined to _not_ be included. Again, probably should be removed and moved to another Pr, but idk how much pain that will be

Reply:
I'd much rather this was in another PR. It's adding noise. Are these changes required for this PR to work?

Reply:
Reverted